django/db/backends/oracle/creation.py

```diff
@@ -12,9 +13,26 @@
 
 class DatabaseCreation(BaseDatabaseCreation):
 
+    @cached_property
+    def _maindb_connection(self):
+        """
+        This is analogous to other backends' `_nodb_connection` property,
+        which allows access to an "administrative" connection which can
+        be used to manage the test databases.
+        For Oracle, the only connection that can be used for that purpose
+        is the main (non-test) connection.
+        """
+        settings_dict = settings.DATABASES[self.connection.alias]
+        user = settings_dict.get('SAVED_USER') or settings_dict['USER']
```

Comment:
separate issue but I have no idea what the existing SAVED_USER/SAVED_PASSWORD stuff is and it doesn't seem to be explained anywhere